docs/docs/behaviors/lighting.md

```diff
@@ -42,10 +43,31 @@ The "RGB underglow" behavior completes an RGB action given on press.
 ### Behavior Binding
 
 - Reference: `&rgb_ug`
-- Parameter: The RGB action define, e.g. `RGB_TOG` or `RGB_BRI`
+- Parameter #1: The RGB action define, e.g. `RGB_TOG` or `RGB_BRI`
+- Parameter #1: Only applies to `RGB_COLOR_HSB` and is the HSB values of the color to set within parenthesis and separated by a common (see below for an example)
```

Comment:
The way `RGB_COLOR_HSB` is used doesn't really seem like a two parameter system to the user due to the macro. I'm not sure how this can be worded better, but the example definitely shows it well.

Reply:
It may not 'feel right' but it's in line with the rest of the documentation and how the underlying behavior system sees things. With the example I think this will be OK.

Reply:
Fair enough. I'd correct it to say Parameter #2 at least 👍